clightning/clightning_commands.go

```diff
@@ -651,14 +639,18 @@ func (l *ListPeers) Call() (jrpc2.Result, error) {
 				PaidFee: paidFees,
 			}
 
-			peerSwapPeerChannels := []*PeerSwapPeerChannel{}
+			peerSwapPeerChannels := []*peerswaprpc.PeerSwapPeerChannel{}
 			for _, channel := range peer.Channels {
 				if c, ok := fundingChannels[channel.ShortChannelId]; ok {
-					peerSwapPeerChannels = append(peerSwapPeerChannels, &PeerSwapPeerChannel{
-						ChannelId:     c.ShortChannelId,
+					scid, err := peerswaprpc.NewScidFromString(c.ShortChannelId)
+					if err != nil {
+						return nil, err
+					}
+					peerSwapPeerChannels = append(peerSwapPeerChannels, &peerswaprpc.PeerSwapPeerChannel{
+						ChannelId:     scid.ToUint64(),
```

Comment:
I think there is a concern that replacing the channel id with a short channel id would be a breaking change and have a impact.
I don't think there will be a problem if both channel id and short channel id are set, so I decided to just add a short channel id.

Reply:
Internally we use the scid so there should no be a problem here. Changing from `channel_id` to `scid` would "break" the api that is just consumed by `peerswapcli` and maybe RTL so far. Vice versa for a change from `scid` to `channel_id` on the core-lightning plugin. We either way have to go with on change on the user facing api :sweat_smile:

Reply:
Sorry, this may have caused a little confusion, so I will state it again.

I considered the following two way.
I personally believe that `1` is the way that will have the least impact, which do you think better?

1. Add `string short_channel_id = 6;`.  
The current change(https://github.com/ElementsProject/peerswap/commit/de30b187a74239a7cab2cfc571edd4fdb1b11f41) does this.
```proto
message PeerSwapPeerChannel {
    uint64 channel_id = 1;
    uint64 local_balance = 2;
    uint64 remote_balance = 3;
    bool active = 5;
    string short_channel_id = 6;
}
```

2. Update uint64 `uint channel_id = 1;` to `string short_channel_id = 1;`
```proto
message PeerSwapPeerChannel {
    string short_channel_id = 1;
    uint64 local_balance = 2;
    uint64 remote_balance = 3;
    bool active = 5;
}
```